src/index.scss

```diff
@@ -162,84 +158,68 @@
 	pointer-events: none;
 }
 
+// Inserter block type icon
 .block-editor-block-types-list__item-icon {
 	scale: 1.3;
 }
 
+// Inserter block type title
 .block-editor-block-types-list__item-title {
 	font-size: 17px;
 }
 
 /* Settings and more */
 
-.block-editor-block-card__title {
-	font-size: 17px;
-	font-weight: 600;
-}
-
+// Block settings description
 .block-editor-block-card__description {
 	font-size: 15px !important;
 	margin-top: -4px !important;
 }
 
+// Block settings heading
 .block-editor-block-inspector h2 {
 	font-size: 17px !important;
 	font-weight: 600 !important;
 }
 
-.block-editor-block-inspector .block-editor-block-icon {
-	padding-top: 4px;
-	/* padding-right: 0px; */
-	margin-right: 8px;
-}
-
```

Comment:
I deemed these subtle adjustments are not worth the maintenance burden of
manually overriding core styles.